public/js/splitjs/splitjsDir.directive.js

```diff
@@ -2,19 +2,19 @@
 
 module.exports =
 angular
-.module('diSplitJs.directives', [])
-.directive('split-area'), function(){
```

Comment:
@JuanCaicedo here's where I went wrong. Notice that the parens executing the function are misplaced